src/kOS.Safe/Encapsulation/SuffixedList.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
```

Comment:
I like the idea in general of making it so when a user does LIST(), they get a list that can only contain kOS objects and can't contain underlying C# objects that have no kOS wrapper.  But that general idea requires that the FIRST step be to make it so we wrapper around all the primitives like string, float, int, etc.  Then you can make a List that requires its members to be of type kOSObject, where kOSObject is the base all our wrappers derive from.

But this... this is only halfway there and I'm not sure if that's worth the work because it would just have to be gutted and done over again later when the full solution happens.

What was the rationale for WHY we needed a list that is only allowed to contain suffixed items, but don't need one to contain other kOS objects.  What was the use case this was trying to solve?